lastminutewf2020.yml

```diff
@@ -75,3 +75,41 @@
         msg: "Finished {{ version }}"
         priority: "notice"
       changed_when: false
+    - name: workaround nautilus auto-mounting
```

Comment:
I think all of these should before the finished log entry on L75

Reply:
done.